spec/crypto.spec.ts

```diff
@@ -1,3 +1,5 @@
+const { Blob } = require('blob-polyfill')
```

Comment:
Use `import { Blob } from 'blob-polyfill'` syntax -- it will be transpiled to require syntax.

Reply:
> Could not find a declaration file for module 'blob-polyfill'. '/Users/gt-liangy/Developer/formsg-sdk/node_modules/blob-polyfill/Blob.
js' implicitly has an 'any' type.

I'm facing the above error... do I really have to write a custom typedef file?